src/main/java/strategies/impl/AllStrategyImpl.java

```diff
@@ -0,0 +1,34 @@
+package strategies.impl;
+
+import strategies.SearchStrategy;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class AllStrategyImpl implements SearchStrategy {
+    List<Integer> wordIndexes;
```

Comment:
I think that's a redundant variable. Can you please remove it?

Reply:
removed this